metadata-ingestion/src/datahub/cli/check_cli.py

```diff
@@ -22,6 +26,33 @@ def mce_file(json_file: str) -> None:
     click.echo(report)
 
 
+@check.command(hidden=True)
+@click.argument("json-file", type=click.Path(exists=True, dir_okay=False))
+def canonicalize_mcp_file(json_file: str) -> None:
```

Comment:
how about adding this as a flag to the existing check mce_file command?
maybe we should call it a different name in the light of mce -> mcp:
`datahub check metadata_file --json-file <metadata_events.json> --lintFix`